File: module_novel/src/main/java/com/yyxnb/module_novel/ui/NovelActivity.java
package com.yyxnb.module_novel.ui;

import android.support.v4.app.Fragment;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.yyxnb.common_base.core.ContainerActivity;

import static com.yyxnb.common_res.arouter.ARouterConstant.NOVEL_MAIN;

@Route(path = NOVEL_MAIN)
public class NovelActivity extends ContainerActivity {

    @Override
    public Fragment initBaseFragment() {
        return new NovelMainFragment();
    }
}

File: module_novel/src/main/java/com/yyxnb/module_novel/ui/NovelActivity.kt
package com.yyxnb.module_novel.ui

import androidx.fragment.app.Fragment
import com.alibaba.android.arouter.facade.annotation.Route
import com.yyxnb.common_base.arouter.ARouterConstant.NOVEL_MAIN
import com.yyxnb.common_base.base.ContainerActivity

@Route(path = NOVEL_MAIN)
class NovelActivity : ContainerActivity() {

    override fun initBaseFragment(): Fragment? {
        return NovelMainFragment()
    }
}